### ISSUE TITLE
Integrate a stopwatch directly into the app

### ISSUE THREAD

**Issue by Snouzy** [Vincenius/workout-lol#12]
So the user who has not a watch on his wrist is not forced to go to his "clock" native app. 

Maybe it include to purpose how many rest the user want between sets / exercices ? 

What do you think ?

**Comment by Vincenius** [comment]
Sounds like a great idea!

**Comment by Snouzy** [comment]
We could compute the approximative time of the workout based on rest time (this feature) + sets and reps. 
We could assume 10 reps = approximatly 30s

so the user will know before starting a workout how many time this is gonna take

**Comment by Vincenius** [comment]
Good idea - I also thought of displaying a estimated time when choosing the exercises but didn't know how to compute it.
Would be a great addition once we added the recommended reps / sets

**Comment by jantznick** [comment]
I could look into adding a watch feature. Any ideas how to best implement it? Currently thinking a timer in the browser or just setting a new variable of workout start time and calculating total time by subtracting current time from start time. That may be a bit more work than just a simple in browser timer though.

For now not worried about estimated time to complete, perhaps that can get added once recommended sets is added.
